.github/CODEOWNERS

```diff
@@ -1,17 +1,16 @@
-# Code owners have the ability to approve a pull request that has made changes in the below files or package(s).
-# At least one Code Owner will be required for any package.
+# *Code owners* can approve pull requests for packages they are assigned to below.
```

Comment:
This first sentence is actually inaccurate. *Any* collaborator can approve *any* pull requests, regardless of whether they've been assigned to that PR or not. We can probably remove the first sentence.

Reply:
Deleted